File: livingrimoire start here/LivinGrimoire java/src/Auxiliary_Modules/InputFilter.java
package Auxiliary_Modules;

public class InputFilter {
    // filter out non-relevant input
    // or filter in relevant data
    public String filter(String ear, String skin, String eye){
        // override me
        return "";
    }
    public AXKeyValuePair filter(String ear){
        // override me : key = context/category, value: param
        return new AXKeyValuePair();
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/auxiliary_modules/InputFilter.kt
package auxiliary_modules

class InputFilter {
    // filter out non-relevant input
    // or filter in relevant data
    fun filter(ear: String, skin: String, eye: String): String {
        // override me
        return ""
    }

    fun filter(ear: String): AXKeyValuePair {
        // override me : key = context/category, value: param
        return AXKeyValuePair()
    }
}